# HG changeset patch
# User Ray Donnelly <mingw.android@gmail.com>
# Date 1471128963 -3600
#      Sat Aug 13 23:56:03 2016 +0100
# Node ID dd84aef4c2166c6fbf63fd75dc567d572bda3fae
# Parent  f272d2d75e7a9f6d5568f1471f1d7544a4d3df25
ResetConsole is not a member of structRstart

diff --git a/rpy/rinterface/_rinterface.c b/rpy/rinterface/_rinterface.c
--- a/rpy/rinterface/_rinterface.c
+++ b/rpy/rinterface/_rinterface.c
@@ -1275,7 +1275,7 @@
   Rp->Busy = Re_Busy;
   Rp->ShowMessage = EmbeddedR_ShowMessage;
   /* Rp->FlushConsole = EmbeddedR_FlushConsole; */
-  Rp->ResetConsole = EmbeddedR_ResetConsole; 
+  /* Rp->ResetConsole = EmbeddedR_ResetConsole; */
   Rp->CallBack = win32CallBack;
   Rp->R_Quiet = FALSE;
   Rp->R_Interactive = TRUE;
